presto-product-tests/conf/docker/singlenode-mapr/compose.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+source ${BASH_SOURCE%/*}/../common/compose-commons.sh
+
+docker-compose \
+-f ${BASH_SOURCE%/*}/../common/standard.yml \
+-f ${BASH_SOURCE%/*}/../common/jdbc_db.yml \
+-f ${BASH_SOURCE%/*}/../common/cassandra.yml \
```

Comment:
This isn't going on the sprint branch. It'll stay on the `mapr-support` branch until we figure out what's going to happen.